src/components/Link.js

```diff
@@ -0,0 +1,4 @@
+import { Link } from 'redux-little-router';
```

Comment:
Ohhhhhhhhhhh. Yeah, I'm a bit skeptical as well if that's what is happening. If we do go this route, I'd suggest just dropping this as a comment in these components.